cn-india/middlewares/isAuth.js

```diff
@@ -0,0 +1,27 @@
+const express = require("express");
+const User = require("../models/user");
+var jwt = require("jsonwebtoken");
+
+module.exports = async (req, res,next) => {
+  const authorization = req.get("Authorization");
+  const token = authorization.split(' ')[1];
+  // console.log(token)
+ 
+  const decode_token = jwt.decode(token, "mysecretkey");
```

Comment:
wrap line no. 10 in try block because jwt.verify will throw error if wrong signature is given